<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1358</width>
    <height>851</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1246</width>
    <height>783</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(221, 237, 220);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>1201</width>
        <height>650</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="registerWidget_1">
       <widget class="QFrame" name="registerFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>1201</width>
          <height>650</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>1201</width>
          <height>650</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1147</width>
          <height>578</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
        background-color: rgb(42, 45, 55);
        padding: 20px 20px 20px 40px;
        border-radius: 50px;
}

QLineEdit {
        border-bottom: 2px solid gray;
        background-color: rgb(42, 45, 55);
        padding: 10px 20px 10px 20px;
        font-size: 11pt;
        border-radius: 10px;
        margin-top: 30px;
    color: rgb(226, 226, 226);
}

QLineEdit:hover {
        border-color: #ffffff;
}

QPushButton {
        padding: 10px 25px 10px 25px;
        border: 2px solid gray;
        border-radius: 10px;
        background-color: rgb(125, 197, 120);
        color:#ffffff;
        font: 600 italic 14pt &quot;Cascadia Mono&quot;;

}
QPushButton:hover {
        background-color: rgb(76, 120, 73);
        border-radius: 10px;

}

QLabel {
        padding: 10px 40px 10px 10px;
    color: white;
}

QPushButton:disabled {
        background-color: rgb(168, 197, 163);
}

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item row="9" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="nextButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Next ⏭️</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLineEdit" name="firstName">
                 <property name="placeholderText">
                  <string>First Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lastName">
                 <property name="placeholderText">
                  <string>Last Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="username">
                 <property name="placeholderText">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="usernameLabel">
                 <property name="text">
                  <string>⚠️ Your username is already taken.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="email">
                 <property name="styleSheet">
                  <string notr="true">margin-top: 10px</string>
                 </property>
                 <property name="placeholderText">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
              <widget class="QLabel" name="registerHeading">
               <property name="styleSheet">
                <string notr="true">color: #ffffff;
font: 600 italic 16pt &quot;Cascadia Mono&quot;;</string>
               </property>
               <property name="text">
                <string>Join our Community !</string>
               </property>
              </widget>
             </item>
             <item row="11" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="emailLabel">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>⚠️ Please enter a valid email.</string>
               </property>
               <property name="textFormat">
                <enum>Qt::PlainText</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="10" column="0">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="registerIllustration">
             <property name="minimumSize">
              <size>
               <width>580</width>
               <height>500</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #ffffff;
margin-left: 50px;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/UserValidation/media/rb_7853.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="registerWidget_2">
       <widget class="QFrame" name="emailFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>1201</width>
          <height>650</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>1201</width>
          <height>650</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1201</width>
          <height>650</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
        background-color: rgb(42, 45, 55);
        padding: 20px 20px 20px 40px;
        border-radius: 50px;
}

QLineEdit {
        border-bottom: 2px solid gray;
        background-color: rgb(42, 45, 55);
        padding: 10px 20px 10px 20px;
        font-size: 11pt;
        border-radius: 10px;
        margin-top: 30px;
    		color: rgb(226, 226, 226);
}

QLineEdit:hover {
        border-color: #ffffff;
}

QPushButton {
        padding: 10px 25px 10px 25px;
        border: 2px solid gray;
        border-radius: 10px;
        background-color: rgb(125, 197, 120);
        color:#ffffff;
        font: 600 italic 14pt &quot;Cascadia Mono&quot;;

}
QPushButton:hover {
        background-color: rgb(76, 120, 73);
        border-radius: 10px;
}

#resendButton {
	background-color: transparent;
	border: transparent;
	font: 500 10pt;
	padding: 0px 15px 0px 0px;
}

#resendButton:disabled {
	color: rgb(147, 147, 147)
}

#resendButton:hover {
	text-decoration: underline;
}

QLabel {
        padding: 10px 40px 10px 10px;
   		color: white;
		font: 600 11pt &quot;Fira Code SemiBold&quot;;
}

QPushButton:disabled {
        background-color: rgb(168, 197, 163);
}

#resendtimer {
	 color: rgb(147, 147, 147)
}


</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,3">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
              <widget class="QLabel" name="emailConfirmHeading">
               <property name="styleSheet">
                <string notr="true">color: #ffffff;
font: 600 italic 16pt &quot;Cascadia Mono&quot;;</string>
               </property>
               <property name="text">
                <string>Please check your inbox for confirmation !</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="7" column="0">
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="resendTimer">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="resendButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Resend</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <widget class="QLineEdit" name="emailConfirmCode">
               <property name="inputMask">
                <string/>
               </property>
               <property name="maxLength">
                <number>6</number>
               </property>
               <property name="placeholderText">
                <string>6-Digit Code</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="changeEmailButton">
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Change Email</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="5" column="0">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="8" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_2"/>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="emailConfirmIllustration">
             <property name="minimumSize">
              <size>
               <width>580</width>
               <height>500</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #ffffff;
margin-left: 50px;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/UserValidation/media/rb_21298.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionewkljfnwef">
   <property name="text">
    <string>ewkljfnwef''</string>
   </property>
  </action>
  <action name="actionwlkemfm">
   <property name="text">
    <string>wlkemfm</string>
   </property>
  </action>
  <action name="action_kmf">
   <property name="text">
    <string>;kmf</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
